controllers/storageclassrequest/storageclassrequest_controller.go

```diff
@@ -422,6 +423,8 @@ func (r *StorageClassRequestReconciler) reconcileCephFilesystemSubVolumeGroup()
 
 	r.setCephResourceStatus(r.cephFilesystemSubVolumeGroup.Name, "CephFilesystemSubVolumeGroup", phase, cephClients)
 
+	storageProfile.Status.Phase = v1.StorageProfilePhaseReady
```

Comment:
Same as my last comment about updating the status of the storage profile CR in this controller.